### ISSUE TITLE
Native Android app for Memories

### ISSUE THREAD

**Issue by BluePixel4k** [pulsejet/memories#1219]
**Is your feature request related to a problem? Please describe.**
Currently, The Android app uses the system WebView to display the Nextcloud Memories page. Overall, it's working well, but it's not as smooth as a natively programmed Android app. Because it's a bit slower and some things are a bit buggy.

**Describe the solution you'd like**
A native Android app for Memories in the future would be a great improvement.
I know that this means a lot more effort - therefore, it's a nice-to-have feature.

**Additional context**
Thank you for your work! :)

**Comment by zeroheure** [comment]
it already exist
https://f-droid.org/packages/gallery.memories/

**Comment by BluePixel4k** [comment]
No, that's not a native Android app. It's only an app, that uses the system WebView to display the website inside the app.
